extensions/wikia/JWPlayerTag/JWPlayerTagController.class.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+class JWPlayerTagController extends WikiaController {
+
+	const PARSER_TAG_NAME = 'jwplayer';
+
+	const ELEMENT_ID_PREFIX = 'jwPlayerTag';
+
+	const CLASS_ATTR = 'class';
+	const COMPONENT_ATTR = 'data-component';
+	const DATA_ATTRS = 'data-attrs';
+	const DATA_MEDIA_ID_ATTR = 'data-media-id';
+	const ID_ATTR = 'id';
+	const STYLE_ATTR = 'style';
+	const WIDTH_ATTR = 'width';
+
+	public function __construct() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this method "__construct" to simply inherit it. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1185)

[project-id]: wikia:app